library/ssl_tls13_keys.c

```diff
@@ -1178,6 +1178,10 @@ static int ssl_tls13_generate_early_key(mbedtls_ssl_context *ssl,
     traffic_keys->key_len = key_len;
     traffic_keys->iv_len = iv_len;
 
+    /* Erase early secrets */
+    mbedtls_platform_zeroize(
+        &tls13_early_secrets, sizeof(mbedtls_ssl_tls13_early_secrets));
+
```

Comment:
```suggestion

```

Reply:
Although `tls13_early_secrets` is a local variable, I think it's better and safer to zeroize the value in `tls13_early_secrets` before its expiration. So the value of `tls13_early_secrets` is kept secret. Does this make sense?

Reply:
It makes sense to me, better to delete secrets from memory as soon as we do not need them anymore. If we do not do it here then we do not control when it is going to be deleted from the stack.

Reply:
Agree that it's a safer way to corrupt the secrets in memory immediately once it's no need anymore. Anyway, maybe we should re-consider the zeroize time in other key-related functions such as  mbedtls_ssl_tls13_generate_application_keys() and mbedtls_ssl_tls13_compute_handshake_transform() in another PR.

Reply:
That's duplicate with line 1198. 

Please remove it